tests/org/intellij/grammar/BnfGeneratorTest.java

```diff
@@ -35,6 +35,9 @@ public BnfGeneratorTest() {
   public void testTokenSequence() throws Exception { doGenTest(false); }
   public void testTokenChoice() throws Exception { doGenTest(true); }
   public void testTokenChoiceNoSets() throws Exception { doGenTest(true); }
+  public void testSimpleSealedTypes() throws Exception { doGenTest(true); }
+  public void testSealedTypesMethods() throws Exception { doGenTest(true); }
+  public void testExtendsIsAppliedOverSealed() throws Exception { doGenTest(true); }
```

Comment:
This test tests a lot while the intended behavior under test is only that it generates the same code as bnf without the sealed modifers